src/Sarif/Schemata/sarif-2.1.0-rtm.6.json

```diff
@@ -1709,24 +1709,28 @@
         "startLine": {
           "description": "The line number of the first character in the region.",
           "type": "integer",
+          "default": 0,
```

Comment:
>"default": 0, [](http://example.com/codeflow?start=10&length=13)

for the 4 number properties, we will do in a separated PR.
This change is temporary just for skip them so they will not be generated as nullable, and the PR is against the dev branch, we will get to it before to main, so this change will not show up in main at all.

Reply:
so what's odd about this is we have a default of -1 for other things, why not this?

how can a default be lower than the minimum value? seems odd.